hazelcast-integration/openshift/client-apps/ocp-demo-frontend/src/main/resources/hazelcast-client.xml

```diff
@@ -1,6 +1,6 @@
 <hazelcast-client xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                   xsi:schemaLocation="http://www.hazelcast.com/schema/client-config
-                               http://www.hazelcast.com/schema/client-config/hazelcast-client-config-3.11.xsd"
+                               http://www.hazelcast.com/schema/client-config/hazelcast-client-config-4.0.xsd"
```

Comment:
What about replacing this file by hazelcast-client.yaml then we do not need maintain xsd versions.

Reply:
ok, done